winui/Kanban/getting-started.md

```diff
@@ -0,0 +1,273 @@
+
+To automatically generate columns in the Kanban control, you can set the `AutoGenerateColumns` property to `true`. When enabled, the columns are generated dynamically based on the `Category` property of KanbanModel, creating a column for each distinct value in the `ItemsSource`.
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="True">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+this.kanban.AutoGenerateColumns = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Manually defining columns in the Kanban control
+
+You can manually define columns in the Kanban control by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban. The `ItemsSource` bound to the Kanban control will populate the respective columns based on the `Categories` property in `KanbanColumn`.
+
+The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 4 5 6" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="False">
+    <kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
+    <kanban:KanbanColumn Categories="In Progress" HeaderText="Progress"/>
+    <kanban:KanbanColumn Categories="Review,Closed" HeaderText="Done"/>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines=" 2 4 5 6" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+this.kanban.AutoGenerateColumns = false;
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", });
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "In Progress", HeaderText = "Progress", });
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Review,Closed", HeaderText = "Done", });
+
+{% endhighlight %}
+{% endtabs %}
+
+N> When manually defining columns, ensure the `AutoGenerateColumns` property of `SfKanban` is set to `false`.
+
+## Theme
+
+The WinUI Kanban supports light and dark themes, automatically adjusting to the system's theme settings for a consistent and visually appealing experience in any environment.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
+
+N> You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Examples` that shows you how to render the Kanban in WinUI.
+
+Also, you can find the Kanban keys for the Light, Dark, and High Contrast themes here.
```

Comment:
Remove this line, add our resource keys in https://github.com/syncfusion/winui-controls-theme-resource-files